api/src/services/permit-service.test.ts

```diff
@@ -1,313 +1,10 @@
-    });
-
-    it('should throw an error when a failure occurs', async () => {
-      const expectedError = new Error('cannot process request');
-
-      const mockDBConnection = getMockDBConnection();
-
-      const permitService = new PermitService(mockDBConnection);
-
-      sinon.stub(PermitService.prototype, 'insertNoSamplePermit').rejects(expectedError);
-
-      try {
-        await permitService.createNoSamplePermits(sampleReq.body);
-
-        expect.fail();
-      } catch (actualError) {
-        expect((actualError as HTTPError).message).to.equal(expectedError.message);
-      }
-    });
-  });
-
-  describe('insertNoSamplePermit', () => {
-    afterEach(() => {
-      sinon.restore();
-    });
-
-    const permitData = {
-      permit_number: 'number',
-      permit_type: 'type'
-    };
-
-    const coordinatorData = {
-      first_name: 'first',
-      last_name: 'last',
-      email_address: 'email@example.com',
-      coordinator_agency: 'agency',
-      share_contact_details: true
-    };
-
-    it('should throw an error when cannot generate post sql statement', async () => {
-      const mockDBConnection = getMockDBConnection();
-
-      sinon.stub(permit_queries, 'postPermitNoSamplingSQL').returns(null);
-
-      const permitService = new PermitService(mockDBConnection);
-
-      try {
-        await permitService.insertNoSamplePermit(permitData, coordinatorData);
-
-        expect.fail();
-      } catch (actualError) {
-        expect((actualError as HTTPError).status).to.equal(400);
-        expect((actualError as HTTPError).message).to.equal('Failed to build SQL insert statement');
-      }
-    });
-
-    it('should throw a HTTP 400 error when failed to insert non-sampling permits cause result is null', async () => {
-      const mockQueryResponse = (null as unknown) as QueryResult<any>;
-      const mockDBConnection = getMockDBConnection({ query: async () => mockQueryResponse });
-
-      sinon.stub(permit_queries, 'postPermitNoSamplingSQL').returns(SQL`some`);
-
-      const permitService = new PermitService(mockDBConnection);
-
-      try {
-        await permitService.insertNoSamplePermit(permitData, coordinatorData);
-
-        expect.fail();
-      } catch (actualError) {
-        expect((actualError as HTTPError).status).to.equal(400);
-        expect((actualError as HTTPError).message).to.equal('Failed to insert non-sampling permit data');
-      }
-    });
-
-    it('should return the result id on success', async () => {
-      const mockQueryResponse = ({ rows: [{ id: 12 }] } as unknown) as QueryResult<any>;
-      const mockDBConnection = getMockDBConnection({ query: async () => mockQueryResponse });
-
-      sinon.stub(permit_queries, 'postPermitNoSamplingSQL').returns(SQL`some`);
-
-      const permitService = new PermitService(mockDBConnection);
-
-      const res = await permitService.insertNoSamplePermit(permitData, coordinatorData);
-
-      expect(res).to.equal(12);
-    });
+  it('TODO', () => {
```

Comment:
could be removed

Reply:
Removed, and added real tests (was just a placeholder so as not to forget)